playbooks/pas_gcp_dev.yml

```diff
@@ -0,0 +1,20 @@
+---
+- hosts: pas_gcp_dev
```

Comment:
In order to add name here we will need a routable IP address. We can't get a routable IP address from our Google Cloud VMs. They all have [RFC 1918](https://datatracker.ietf.org/doc/html/rfc1918) so if I enter a name it will fail. The connection goes through a bastion host.